pipeline/src/handler.ts

```diff
@@ -9,12 +9,32 @@ import {
   webcomicsQuery,
   wrapQueries,
 } from './graph-queries';
+// import { addressablesQuery } from './graph-queries/addressables';
 import { articles, events, venues } from './indices';
+// import { transformAddressable } from './transformers/addressables';
 import { transformArticle } from './transformers/article';
 import { transformEventDocument } from './transformers/eventDocument';
 import { transformVenue } from './transformers/venue';
 import { Clients } from './types';
 
+// const loadAddressables = createETLPipeline({
+//   graphQuery: addressablesQuery,
+//   indexConfig: addressables,
+//   parentDocumentTypes: new Set([
+//     'articles',
+//     'books',
+//     'events',
+//     'exhibitions',
+//     'exhibition-texts',
+//     'exhibition-highlight-tours',
+//     'pages',
+//     'projects',
+//     'seasons',
+//     'visual-stories',
+//   ]),
+//   transformer: transformAddressable,
+// });
+
```

Comment:
As discussed; We could uncomment this, but then I'd say we only put in the content types who have transformers